app/models/metrics.rb

```diff
@@ -33,22 +33,22 @@ def initialize(root, group_by: nil, time_period:)
   alias :read_attribute_for_serialization :send
   attr_reader :group_by, :root, :time_period
 
-  def metrics
```

Comment:
This is unrelated to your changes, but now that I re-read this, `metrics` isn't a very clear name. I wonder what else we can rename this to, `totals` perhaps?

Reply:
I made this change, but this introduces a dependency between this PR and a view PR, which might be best done separately as they're already quite dependent.  I'll remove it and we can do the change in another PR.

Reply:
Yeah, of course. This is probably overkill, as we're not in production, but one way to decouple this would be to include both `metrics` and `totals` in the serialised object. And then remove it in a later commit.